openlibrary/catalog/marc/build_record.py

```diff
@@ -43,7 +46,11 @@
 
 re_series = re.compile('^(.*) series$', re.I)
 
+def deprecated():
```

Comment:
This would be a good place to use the Deprecated module which provides this as a decorator and also allows one to specify a reason (and version, but we don't have versions). 
https://pypi.org/project/Deprecated/

Reply:
Good idea, I made the change locally but the deprecated module needs to be installed, and will need to be on production. @mekarpeles do we have a process for adding new modules conveniently to prod?

I have been finding these deprecation warnings helpful in tracking and removing old code. The decorator makes this easier to do.